Lib/idlelib/query.py

```diff
@@ -224,6 +224,25 @@ def entry_ok(self):
         return file_path
 
 
+class Goto(Query):
+    "Get a positive line number for editor Go To Line."
+    # Used in editor.EditorWindow.goto_line_event.
+
+    def __init__(self, parent, title, message, *, _htest=False, _utest=False):
```

Comment:
This `__init__` seems unnecessary.

Reply:
Right, I copied and edited from an existing subclass that does need it.  Deleted.